packages/react-badge/src/components/CounterBadge/useCounterBadgeStyles.ts

```diff
@@ -0,0 +1,73 @@
+import { makeStyles, ax } from '@fluentui/react-make-styles';
+import { CounterBadgeState } from './CounterBadge.types';
+import { useRootStyles, useIconStyles } from '../Badge/useBadgeStyles';
+
+/**
+ * Styles for the root slot
+ */
+export const useCounterBadgeRootStyles = makeStyles<CounterBadgeState>([
+  [
+    null,
+    {
+      minWidth: 'auto',
+    },
+  ],
+  [
+    s => s.color === 'warning',
+    theme => ({
+      backgroundColor: theme.global.palette.cranberry.primary,
+      borderColor: theme.global.palette.cranberry.primary,
+    }),
```

Comment:
All of these should be setting `color` as well, right?

Reply:
Well, actually no. From the design they are just white that is the default